imageflow/models.py

```diff
@@ -174,6 +175,15 @@ def get_short_name(self, maxlen=30):
         n_1 = maxlen - n_2 - 3
         return '%s...%s' % (name[:n_1], name[-n_2:])
 
+    def save(self, user, *args, **kwargs):
```

Comment:
I think it would actually make sense to approach this problem different way.  I don't think we should overload the `save()` method because:

1) It is a very core method and a lot of existing code relies on its typical method signature (eg. Django plugins and Astrokit code outside the views).  It's quite possible future developers could forget to add the parameter to their save calls.
2) We'd have to overload the method for other editable objects too, like `Reduction` and `LightCurve`
3) This only could prevent unauthorized writes, not unauthorized reads (I know that was not a requirement for this project, but it explains why this is not a typical approach to authorization as far as I know)

Because there is already a `user` attribute in this model, I think it would be better to just check the user when we fetch the light curve in the view logic, not the model logic.  I'll comment on the relevant code below.